projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -74,8 +93,26 @@
     border: tokens.$cds-alias-object-border-width-100 solid datepicker-variables.$clr-calendar-border-color;
     border-radius: tokens.$cds-alias-object-border-radius-100;
     box-shadow: tokens.$cds-alias-object-shadow-100;
-    overflow: hidden;
     z-index: map.get(variables.$clr-layers, dropdown-menu);
+
+    &.has-action-buttons {
+      height: calc(
+        datepicker-variables.$clr-calendar-height + datepicker-variables.$clr-calendar-action-buttons-height
+      );
+    }
+
+    .datepicker-actions {
+      min-width: 100%;
+      height: datepicker-variables.$clr-calendar-action-buttons-height;
+      display: flex;
+      border-top: tokens.$cds-global-space-1 solid tokens.$cds-alias-object-container-border-color;
+      justify-content: end;
+      padding-top: tokens.$cds-global-space-7;
+      gap: button-variables.$clr-btn-horizontal-margin;
+      button {
```

```suggestion
      gap: button-variables.$clr-btn-horizontal-margin;

      button {
```

nit: Add blank line between blocks.